dashboard/app/mailers/pd/regional_partner_mini_contact_mailer.rb

```diff
@@ -1,4 +1,5 @@
-class Pd::RegionalPartnerMiniContactMailer < ActionMailer::Base
+class Pd::RegionalPartnerMiniContactMailer < ApplicationMailer
+  include ActionMailerMetrics
```

Comment:
You don't need this since it's already included in the parent class `ApplicationMailer`